Commands/Publishing/AddHtmlPublishingPageLayout.cs

```diff
@@ -7,6 +7,10 @@ namespace SharePointPnP.PowerShell.Commands.Publishing
     [Cmdlet(VerbsCommon.Add, "SPOHtmlPublishingPageLayout")]
     [CmdletHelp("Adds a HTML based publishing page layout",
        Category = CmdletHelpCategory.Publishing)]
+    [CmdletExample(
+        Code = @"PS:> Add-SPOHtmlPublishingPageLayout -Title 'Our custom page layout' -SourceFilePath 'customlayout.aspx' -Description 'A custom page layout' -AssociatedContentTypeID 0x01010901",
+        Remarks = "Uploads the pagelayout 'customlayout.aspx' to the current site as a 'web part page' pagelayout",
```

Comment:
maybe important to make clear that the file is upload from the current location, like from the current folder.